src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -51,88 +47,94 @@
                                                   [community/community-actions item])}])}
        item])))
 
-(defn render-featured-fn [community-item]
-  (let [item (merge community-item
-                    (get mock-community-item-data :data)
-                    {:featured true})]
-    [community-card/community-card-view-item item #(rf/dispatch [:navigate-to :community-overview item])]))
-
-(defn get-item-layout-js [_ index]
-  #js {:length 64 :offset (* 64 index) :index index})
-
 (defn screen-title []
   [rn/view
    {:height             56
-    :padding-vertical   12
-    :padding-horizontal 20}
+    :padding-vertical   12}
    [quo/text {:accessibility-label :communities-screen-title
               :weight              :semi-bold
               :size                :heading-1}
     (i18n/label :t/discover-communities)]])
 
-(defn featured-communities [communities]
+(defn featured-communities-header [communities-count]
+  [rn/view {:flex-direction     :row
+            :height             30
+            :margin-bottom 8
+            :justify-content    :space-between}
+   [rn/view {:flex-direction :row
+             :align-items    :center}
+    [quo/text {:accessibility-label :featured-communities-title
+               :weight              :semi-bold
+               :size                :paragraph-1
+               :style               {:margin-right 6}}
+     (i18n/label :t/featured)]
+    [quo/counter {:type :grey} communities-count]]
+   [quo/icon :i/info {:container-style {:align-items     :center
+                                        :justify-content :center}
+                      :resize-mode     :center
+                      :size            20
+                      :color           (colors/theme-colors
+                                        colors/neutral-50
+                                        colors/neutral-40)}]])
+
+(defn featured-list [communities view-type]
+  (let [view-size (reagent/atom 0)]
+    (fn []
+      [rn/view {:style {:flex-direction :row
+                        :overflow :hidden
+                        :width "100%"
+                        :margin-bottom 24}
+                :on-layout #(swap! view-size
+                                   (fn []
+                                     (oops/oget % "nativeEvent.layout.width")))}
+       (when-not (= @view-size 0)
+         [rn/flat-list
+          {:key-fn                            :id
+           :horizontal                        true
+           :keyboard-should-persist-taps      :always
+           :shows-horizontal-scroll-indicator false
+           :separator                  [rn/view {:margin-right 12}]
```

Comment:
thars weird separator

Reply:
why so? it's a horizontal scroll in the designs

Reply:
oh, then it could be like [rn/view {:width 12}]] because an empty separator with only a margin looks strange, yeah but anyway, now i know its like a gap between items

Reply:
okay, yeah that also makes sense.